balena/models/device.py

```diff
@@ -835,11 +836,14 @@ def get_status(self, uuid):
 
         """
 
-        device = self.get(uuid)
+        device = self.get_with_service_details(uuid)
+        if not device['is_active']:
+            return DeviceStatus.INACTIVE
```

Comment:
I think this new status should be added to the docstring as well (line 831)

Reply:
@xginn8, updated. Thanks!